app/src/main/java/org/simple/clinic/addidtopatient/searchforpatient/AddIdToPatientSearchScreen.kt

```diff
@@ -72,15 +60,7 @@ class AddIdToPatientSearchScreen(context: Context, attrs: AttributeSet) : Relati
   }
 
   private val allPatientsInFacilityView by unsafeLazy {
-    val (layoutResId, architecture) = if (mobiusMigrationConfig.useAllPatientsInFacilityView) {
-      R.layout.view_allpatientsinfacility to MOBIUS
-    } else {
-      R.layout.view_allpatientsinfacility_old to ORIGINAL
-    }
-
-    Analytics.reportArchitectureMigration("AllPatientsInFacilityView", architecture, "AddIdToPatientSearchScreen")
-
-    allPatientsViewStub.layoutResource = layoutResId
+    allPatientsViewStub.layoutResource = R.layout.view_allpatientsinfacility
     allPatientsViewStub.inflate() as ExposesUiEvents
```

Comment:
We no longer need the `ExposedUiEvents` interface since that was meant to handle the fact that there are two different views, right?